backend/src/test/java/de/neuefische/ffmjava221/teamprojekt/backend/guest/GuestIntegrationTest.java

```diff
@@ -151,30 +152,59 @@ void putRequestUpdateGuestMethodNotAllowed() throws Exception {
         mvc.perform(MockMvcRequestBuilders.put("/api/guests/")
                         .contentType(MediaType.APPLICATION_JSON)
                         .content("""
-                            {"firstName": "test",
-                            "lastName": "test",
-                            "email": "test@gmail.com",
-                            "password": "test",
-                            "confirmPassword": "test",
-                            "id" :  "<id>"}
-                                """))
+                                {"firstName": "test",
+                                "lastName": "test",
+                                "email": "test@gmail.com",
+                                "password": "test",
+                                "confirmPassword": "test",
+                                "id" :  "<id>"}
+                                    """))
                 .andExpect(status().isMethodNotAllowed());
     }
+
     @Test
     @DirtiesContext
     void putRequestUpdateGuestNotFound() throws Exception {
         mvc.perform(MockMvcRequestBuilders.put("/api/guests/1337")
                         .contentType(MediaType.APPLICATION_JSON)
                         .content("""
-                            {"firstName": "test",
-                            "lastName": "test",
-                            "email": "test@gmail.com",
-                            "password": "test",
-                            "confirmPassword": "test",
-                            "id" :  "1337"}
-                                """))
+                                {"firstName": "test",
+                                "lastName": "test",
+                                "email": "test@gmail.com",
+                                "password": "test",
+                                "confirmPassword": "test",
+                                "id" :  "1337"}
+                                    """))
                 .andExpect(status().isNotFound());
     }
+
+    @Test
+    @DirtiesContext
+    void deleteGuestByIdIsSuccessful() throws Exception {
+
+        String body = mvc.perform(MockMvcRequestBuilders.post("/api/guests")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("""
+                                {"firstName": "test",
+                                 "lastName": "test",
+                                 "email": "test@gmail.com",
+                                "password": "test",
+                                 "confirmPassword": "test"}
+                                """))
+                .andExpect(status().isOk())
+                .andReturn().getResponse().getContentAsString();
+
+        Guest guest = objectMapper.readValue(body, Guest.class);
+
+        mvc.perform(MockMvcRequestBuilders.delete("/api/guests/"+guest.id())).andExpect(status().isOk());
+    }
```

Comment:
Now that's not the only thing u expect. U expect Status Code 200 as well as the deleted json object =)